### PR TITLE
Feature: CLI -b/--bind and -p/--port

### PR DESCRIPTION
**Pull request by hosseinsokhandan**

### CODE REVIEW

File: chttpd.c [hunk 1]
```diff
@@ -57,6 +58,61 @@ request_handler(void *cls, struct MHD_Connection *connection, const char *url,
...
+    *port = atoi(port_arg);
+}
+
+
+void
+parse_bind_addr(const char *bind_addr_arg, char *bind_addr, int *port) {
+    int colon_pos = -1;
+
+    if (strcmp(bind_addr_arg, "localhost") == 0) {
+        bind_addr_arg = "127.0.0.1";
```
Comment (hosseinsokhandan) [inline chttpd.c:86]: @pylover please ensure it is good enough to only convert `localhost` to its relevant IP address.

**Comment by pylover** [comment]
I preffer the title: `Feature: CLI -b/--bind and -p/--port`

File: chttpd.c [hunk 2]
```diff
@@ -18,7 +18,8 @@
  */
 #include <stdlib.h>
 #include <unistd.h>
-#include <sys/timerfd.h>
+#include <getopt.h>
```
Comment (pylover) [inline chttpd.c:21]: As told you before, use `GNU ARGP`

**Review by pylover — CHANGES_REQUESTED** [review]

**Comment by pylover** [comment]
https://www.gnu.org/software/libc/manual/html_node/Argp.html

**Review by pylover — APPROVED** [review]

**Pull request merged**